packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/aspnet/enrollmentintegrationsteps.md

```diff
@@ -66,6 +66,7 @@ catch (OktaException exception)
 ```
 
 ### 4: Display a list of possible authenticator factors
```

Comment:
@susanharper-okta @robertnicolo-okta But it's an ordered set of steps and numbering them allows us to map those steps to what is going on in the flow diagrams, and also to match them between the enrollment and challenge flows. Why wouldn't you number them?

Reply:
They're headings. It's not our style to number headings. And also - that wasn't clear to me at all. So, if this is the route that these guides are taking - perhaps let the reader in on that. Because I didn't even notice. But, maybe I'm just not very observant. haha